sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -697,6 +697,253 @@ public enum TextSentiment
         Mixed = 3,
     }
 }
+namespace Azure.AI.TextAnalytics.Models
```

Comment:
For now, please hide all of these types into internals. With each PR adding the design of each feature, these objects might be public